packages/app-desktop/gui/ConfigScreen/controls/plugins/FilterForPlugins/FilterForPlugins.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { themeStyle } from '@joplin/lib/theme';
+import uuid from '@joplin/lib/uuid';
+
+interface FilterProps {
+    themeId: number;
+    setFilterValue: Function;
+	onSearchButtonClick: Function;
+	filterValue: string;
+}
+
+export default function FilterForPlugins(props: FilterProps) {
+
+	const theme = themeStyle(props.themeId);
+
+	const filterOptions = ['Select plugin category','Most Downloaded','Recommended', 'Newest', 'Built-in', 'All', 'line-separator', 'Appearance', 'Developer tools', 'Productivity', 'Themes', 'Integrations', 'Viewer', 'Search', 'Tags', 'Editor', 'Files', 'Personal knowledge management', 'line-separator', 'Installed', 'Enabled', 'Disabled', 'Outdated'];
+
+	const clearFilterValue = () => {
+		(document.getElementById('filter-select') as HTMLInputElement).value = '';
+		props.onSearchButtonClick();
+	};
+
+	return (
+		<div className="filter-for-plugins" style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }} onClick={event => props.setFilterValue((event.target as HTMLInputElement).value)}>
+			<select name="" id="filter-select" defaultValue={'default'} style={{ backgroundColor: theme.backgroundColor, color: theme.color }}>
+				{filterOptions.map((filterOption) => {
+					if (filterOption === 'Select plugin category') return <option value="default" key={uuid.createNano()} disabled>{filterOption}</option>;
```

Comment:
createNano?? But option keys should be stable if the underlying value didn't change. With this it will re-render every time. Please define a programmatic value as mentioned above and use that as key.